### PR TITLE
UI support for target tracking scaling

### PR DESCRIPTION
**Pull request by knguyen100000010**
### Add new target tracking policy

Screenshot 2024-02-07 at 2 10 36 PM

### Display and update existing policies

Screenshot 2024-02-07 at 2 10 27 PM

### CODE REVIEW

File: deploy-board/deploy_board/webapp/group_view.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import json
 import logging
 import traceback
+from itertools import groupby
```
Comment (gzpcho) [inline deploy-board/deploy_board/webapp/group_view.py:26]: Unused import?

**Review by gzpcho — APPROVED** [review]

**Comment by gzpcho** [comment]
lgtm

**Comment by gzpcho** [comment]
Can probably work on making the UI prettier over time

File: deploy-board/deploy_board/webapp/group_view.py [hunk 2]
```diff
@@ -489,16 +489,72 @@ def get_policy(request, group_name):
...
-        if policyType == "simple-scaling":
+        if policyType == "target-tracking-scaling":
+            scaling_policies["scalingPolicies"] = []
+
+            if "content" in params:
+                # update existing policies
+                content = params["content"]
+                input = dict(kv.split("=", 1) for kv in content.split("&"))
+                policy_names = {key: val for key, val in input.items()
+                    if key.startswith("targetScalingName_")}
```
Comment (tylerwowen) [inline deploy-board/deploy_board/webapp/group_view.py:536]: You can probably get the URL prams from `request.GET`

File: deploy-board/deploy_board/templates/groups/asg_policy.tmpl [hunk 3]
```diff
@@ -272,118 +293,319 @@
...
+                                {% csrf_token %}
+                            </form>
+                        {% endif %}
+
+                        <script>
+                            $(function() {
+                                $('#deleteItem_{{policy.policyName}}').click(function () {
+                                            var btn = $(this);
+                                            $.ajax({
+                                                type: 'POST',
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/groups/asg_policy.tmpl:367]: DELETE?

**Review by tylerwowen — COMMENTED** [review]
Left some comments

**Pull request merged**